en/docs/install-and-setup/setup/kubernetes-deployment/kubernetes/am-pattern-5-all-in-one-gw-km.md

```
@@ -40,10 +40,10 @@ This document provides step-by-step instructions to deploy WSO2 API Manager in a
     - [3. Key Manager Configuration](#3-key-manager-configuration)
         - [3.1 Configure Eventhub](#31-configure-eventhub)
         - [3.2 Deploy Key Manager](#32-deploy-key-manager)
-    - [4. Universal Gateway Configuration](#4-universal-gateway-configuration)
+    - [4. Classic Gateway Configuration](#4-universal-gateway-configuration)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)